### PR TITLE
feat(ember-flight-icons): update blueprint

### PR DESCRIPTION
**Pull request by aklkv**
### :pushpin: Summary

Preparatory PR for https://github.com/hashicorp/design-system/pull/2170

(context: https://hashicorp.slack.com/archives/C06NMSYDZPA/p1717440951790419?thread_ts=1717099148.439879&cid=C06NMSYDZPA)

***

### 👀 Component checklist

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: packages/ember-flight-icons/src/components/flight-icon.js
Comment (didoo) [inline packages/ember-flight-icons/src/components/flight-icon.js:19]: I've left a comment in the other PR about this change: https://github.com/hashicorp/design-system/pull/2126#discussion_r1625652230

**Review by didoo — COMMENTED** [review]
I've had a quick look, but my knowledge about these settings is _very_ limited, so I'll defer to @alex-ju to do a in depth review.

File: packages/ember-flight-icons/package.json
Comment (didoo) [inline packages/ember-flight-icons/package.json:68]: @aklkv all the other `package.json` files are pointing to `^2.8.8`, can we keep them consistent to avoid having different behaviours in different parts of the monorepo? it may be confusing.

```suggestion
    "prettier": "^2.8.8",
```

File: packages/ember-flight-icons/.prettierrc.cjs
Comment (didoo) [inline packages/ember-flight-icons/.prettierrc.cjs:4]: @aklkv just to double check: now that we've reverted prettier to `2.8.8` is this change still OK?

**Comment by Dhaulagiri** [comment]
With an eye toward some consistency across the monorepo I excised most of the blueprint-related changes here and got those in https://github.com/hashicorp/design-system/pull/2166. We should be able to rebase this one to get it down to just the TS-related changes and more cleanly review and release that.

**Review by didoo — APPROVED** [review]
I _think_ is OK, but please wait an approval from @alex-ju or @Dhaulagiri before merging

File: packages/ember-flight-icons/package.json
Comment (alex-ju) [inline packages/ember-flight-icons/package.json:87]: now that `template-registry.js` is [a public entrypoint](https://github.com/hashicorp/design-system/pull/2142/files#diff-d6d1229bd915324e84ed0d1ead12d2665b51fb992054efe46249e5988546907bR19) I believe we don't need the alias here anymore.

update: running `yarn build` seems to be removing this line, so we should probably update as such

**Review by alex-ju — APPROVED** [review]
Looking good overall! It took a bit more than expected as I wanted to make sure I test with `atlas` as well

**Pull request merged**
